server/models/Message.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+// Create Schema for Users
+const MessageSchema = new Schema({
+    conversation: {
+        type: Schema.Types.ObjectId,
+        ref: 'conversations',
+    },
+    to: {
+        type: Schema.Types.ObjectId,
+        ref: 'user',
+    },
+    from: {
+        type: Schema.Types.ObjectId,
+        ref: 'user',
+    },
+    body: {
+        type: String,
+        required: true,
+    },
+    createdAt: {
```

Comment:
You can use the timestamps option in mongoose that will automatically generate these for you :)

Reply:
Good idea. This is done